### PR TITLE
[17.0][MIG] mrp_bom_structure_xlsx_level 1

### PR DESCRIPTION
**Pull request by AnaisLForgeFlow**
Standard Migration

@ForgeFlow

### CODE REVIEW

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
The module is available in v16: https://github.com/OCA/manufacture-reporting/tree/16.0/mrp_bom_structure_xlsx_level_1

You need to always migrate from the latest version available.

File: mrp_bom_structure_xlsx_level_1/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This module extends the functionality of the MRP capabilities of Odoo,
+and allows you to export to level 1 of the BoM structure to MS Excel
+.XLSX format.
```
Comment (GuillemCForgeFlow) [inline mrp_bom_structure_xlsx_level_1/readme/DESCRIPTION.md:3]: Perhaps this definition can be improved a little. What do you think about adding this?

> If you have a Bill of Materials (`A`) for which one of the Components also has a Bill of Materials (`B`). The report will only display the Components within the first level for the selected Bill of Materials (`A`), but not the Components from the Bill of Materials (`B`).

**Review by GuillemCForgeFlow — APPROVED** [review]
The rest if perfect. LGTM functional and code review! 👍🏿

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM to me now :+1:

**Pull request merged**